components/schemes/Shape/src/MDCShapeCategory.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "MDCShapeCorner.h"
+
+/**
+ The MDCShapeCategory is the class containing the shape value as part of our shape scheme,
+ MDCShapeScheme.
+
+ MDCShapeCategory is built from 4 corners, that can be set to alter the shape value.
+
+ This takes on the assumption that our settable shapes are made of 4 corners.
```

Comment:
I'm not sure this sentence is communicating any new information (I think it's describing the fact that the class has four corners) - consider removing?